src/main/java/com/codeofus/rent_a_park/models/Reservation.java

```diff
@@ -0,0 +1,42 @@
+package com.codeofus.rent_a_park.models;
+
+import lombok.*;
+import lombok.experimental.FieldDefaults;
+
+import javax.persistence.*;
+import java.time.ZonedDateTime;
+
+@Data
+@Entity
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@ToString
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class Reservation {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    Integer id;
+
+    @ManyToOne
+    @JoinColumn(name = "person_id")
+    Person person;
+
+    @ManyToOne
+    @JoinColumn(name = "spot_id")
+    Spot spot;
+
+    ZonedDateTime createdAt;
```

Comment:
Why ZonedDateTime? Couldn't we convert all to UTC and handle everything as LocalDateTime?